modules/getting-started-web-console-deploying-python-app.adoc

```diff
@@ -6,21 +6,21 @@
 [id="getting-started-web-console-deploying-python-app_{context}"]
 = Deploying a Python application
 
-The following procedure deploys a backend service for the `parksmap` application. The Python application performs 2D geo-spatial queries against a MongoDB database to locate and return map coordinates of all national parks in the world.
+The following procedure deploys a back-end service for the `parksmap` application. The Python application performs 2D geo-spatial queries against a MongoDB database to locate and return map coordinates of all national parks in the world.
 
-The deployed backend service that is `nationalparks`.
+The deployed back-end service that is `nationalparks`.
 
 .Prerequisites
 
 * You are logged in to the {product-title} web console.
-* You are in the xref:../web_console/odc-about-developer-perspective.adoc#odc-about-developer-perspective[*Developer* perspective].
+* You are in the *Developer* perspective.
 * You have a deployed image.
 
 .Procedure
 
 . From the *+Add* view in the *Developer* perspective, click *Import from Git* to open a dialog.
 . Enter the following URL in the Git Repo URL field:
-`https://github.com/openshift-roadshow/nationalparks-py.git`
+`[x-]https://github.com/openshift-roadshow/nationalparks-py.git`
```

Comment:
This is rendering in the doc. the `[x-]` has to go on the outside of the backticks